Dropdown/src/Dropdown.jsx

```diff
@@ -171,30 +171,36 @@ class Dropdown extends Component {
     }
 
     searchItems() {
+
+        const { props } = this;
+
         let closestValueLength = 0, closestValue = null, itemIndex = 0;
 
-        this.props.options.forEach((option, index) => {
+        props.options.forEach((option, index) => {
             let regex = this.state.dropdownText.replace(/([.?*+^$[\]\\(){}|-])/g, "\\$1");
 
+            const label = props.decode ? props.decode(option.label) : option.label;
```

Comment:
Is the check necessary? `decode` is marked as required and has a default value. And what will React do if it is assigned null/undefined?